### PR TITLE
 Remove generate_hash_votes_at from config

### PR DESCRIPTION
**Pull request by SergiySW**
As it's no longer required

### CODE REVIEW

File: rai/node/node.cpp [hunk 1]
```diff
@@ -3819,7 +3806,6 @@ void rai::active_transactions::announce_votes ()
 					else
 					{
 						election_l->compute_rep_votes (transaction);
```
Comment (PlasmaPower) [inline rai/node/node.cpp]: I think this else branch should be removed entirely.

File: rai/node/node.hpp [hunk 2]
```diff
@@ -518,6 +520,11 @@ class node_config
 	rai::stat_config stat_config;
 	rai::uint256_union epoch_block_link;
 	rai::account epoch_block_signer;
+<<<<<<< HEAD
+=======
+	std::chrono::system_clock::time_point generate_hash_votes_at;
+	std::chrono::milliseconds block_processor_batch_max_time;
+>>>>>>> upstream/master
```
Comment (PlasmaPower) [inline rai/node/node.hpp]: Looks like you left in a merge conflict by mistake

**Comment by PlasmaPower** [comment]
It looks like the executable permission of some scripts under utils were removed.

**Comment by SergiySW** [comment]
Weird Windows. Fixed

**Comment by SergiySW** [comment]
It seems common.sh also should be executable

**Comment by PlasmaPower** [comment]
Actually, I believe common.sh was meant to be sourced (and therefore it shouldn't be executable)

**Comment by PlasmaPower** [comment]
LGTM other than that

**Comment by SergiySW** [comment]
Good

**Pull request merged**